src/arm-template/README.md

```diff
@@ -494,3 +496,506 @@ BOX_SERVICE_COSMOSDB_CONNSTR=$(az cosmosdb list-connection-strings \
     /t.ConnectionString:"${BOX_SERVICE_COSMOSDB_CONNSTR};Database=${BOX_DB_NAME};" \
     /t.Collection:Terminals
 ```
+</details>
+
+<details>
+<summary>ENGLISH</summary>
```

Comment:
This part is so big, so please create new document as `src/arm-template/README_en.md` and move this English part to the new document. Then put a link from this README to the README_en.

Reply:
Done